gateway-service/src/main/java/org/zowe/apiml/gateway/routing/ApimlRoutingConfig.java

```diff
@@ -109,4 +112,11 @@ public RoutedInstanceIdFilter routedServerFilter() {
         return new RoutedInstanceIdFilter();
     }
 
+    @Bean
+    @ConditionalOnProperty(name = "instance.metadata.apiml.lb.type", havingValue = "authentication")
+    public PostStoreLoadBalancerCacheFilter postStoreLoadBalancerCacheFilter(AuthenticationService authenticationService,
```

Comment:
This is not correct. The filter is not added to Gateway this way. This should be unconditional, no? This key will not be set on Gateway's configuration.

Reply:
Thanks for catching this. I fixed it in tests but not here.

Reply:
Yes sorry, I was changing the props name but forgot about the fact that it is not part of GW configuration.